### PR TITLE
Require et-al-min to be greater than et-al-use-first (fix)

### PR DESCRIPTION
**Pull request by rmzelle**
Fix small bug in testing of et-al attribute values. (et-al(-subsequent)-min should have a higher value than et-al(-subsequent)-use-first)

(original code from https://github.com/citation-style-language/styles/pull/2207)

### CODE REVIEW

**Comment by rmzelle** [comment]
@adam3smith, just 70 styles! :)

File: rtf-scan.csl [hunk 1]
```diff
@@ -44,7 +44,7 @@
       </else>
     </choose>
   </macro>
-  <citation et-al-min="1" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
+  <citation et-al-min="2" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
```
Comment (rmzelle) [inline rtf-scan.csl]: @adam3smith, I assume this doesn't break anything. Also, any idea whether people actually use this style?

**Comment by rmzelle** [comment]
@adam3smith, I checked the instructions to authors as much as possible for these styles, although I had to guess some styles. Anyway, I think this is good to go. Please squash and merge.

File: rtf-scan.csl (hunk 1 above)
Comment (adam3smith) [inline rtf-scan.csl]: yes, some people do use this and yes, the change is fine.

**Comment by adam3smith** [comment]
Cool, thanks. No concerns from my end. Obviously I didn't double-check the actual et-al values, but I have all confidence that you did this as well or better than I'd be able to.

**Comment by rmzelle** [comment]
Thanks!

**Comment by ukyoi** [comment]
I'm sorry for my rudeness since I'm not a native English speaker, but you really shouldn't just change the file without knowing the details of the corresponding styles.

In Chinese GB/T 7714-1987 and GB/T 7714-2005, the standard specified that the value "et-al-use-first" is ought to be "3". I was happy to find that CSL and used the newest version on my thesis, then got a lot criticism. I know not checking the standard is my fault, but I remember it works fine around one year ago.

**Pull request merged**